src/applications/personalization/dashboard/components/FormItem.jsx

```diff
@@ -34,7 +34,7 @@ class FormItem extends React.Component {
     } = savedFormData.metadata;
     const lastSavedDateTime = moment
       .unix(lastSaved)
-      .format('MMMM D, YYYY [at] h:m a');
+      .format('MMMM D, YYYY [at] h:mm a');
```

Comment:
ooops, the formatting error made it so that  a time like `2:09` would render as `2:9`